hibernate-core/src/main/java/org/hibernate/query/NativeQuery.java

```diff
@@ -505,7 +506,7 @@ interface FetchReturn extends ResultNode {
 
 		String getOwnerAlias();
 
-		String getFetchableName();
+		Fetchable getFetchable();
```

Comment:
Not sure if we consider this to be "API", but wanted to mention the backwards compatibility concern.

Reply:
added it back